wordpress-mobile/WordPressAuthenticator-iOS
# Remove .showEmailLogin segue

Master issue #182 - this is a spike to test removing segues.

This PR removes all references to the segue `.showEmailLogin` and programmatically navigates the user to the `LoginEmailViewController` screen. There are 3 areas where the segue has been removed: WPiOS sign up, WPiOS sign in, and WCiOS sign in.

### To Test - WPiOS
1. Check out this branch
2. Check out this branch for WPiOS: https://github.com/wordpress-mobile/WordPress-iOS/tree/try/break-up-storyboard and point it to your local pod by changing the podfile location to your local copy of WordPressAuthenticator-iOS
3. Build the Authenticator project
4. `rake dependencies` on WPiOS
5. Build the WPiOS project. You are now set up with local pod development! 
6. Navigate to and back from each of these flows: 

1) Login > Continue with WordPress.com - should land on the LoginEmailViewController page
2) Sign up for WordPress.com > Sign up with Email >  Enter an email address that has an existing WP.com account and tap Next - should land on the LoginEmailViewController page

### To Test - WCiOS
1. Check out this branch
2. In the WCiOS project, open the podfile, comment out Authenticator and replace with local pod development. It will look similar to this: `pod 'WordPressAuthenticator', :path => '/Users/thuycopeland/WordPressAuthenticator-iOS/'` 
3. Build the Authenticator project
4. `rake dependencies` on WCiOS
5. Build the WCiOS project. You are now set up with local pod development! 
6. Navigate to and back from this flow: Log in with Jetpack > should show the expected LoginEmailViewController

### To Test - Newspack
When checking out the `develop` branch on Newspack, I see the Site Address flow. I don't think this PR applies to Newspack. Pinging @aerych just in case.
